### DISCUSSION TITLE
Can't see pictures and some other styles when I'm logged in this forum

### DISCUSSION THREAD

**Discussion by imba-tjd** [community/community#21847] [New to GitHub]
<p>Is this expected?</p>
<p>In private mode, the issue doesn’t exist:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/928b4b424c5152e3be22a6924bdbc37a76692345.png" data-download-href="/uploads/short-url/kUo95LHenJWgLTjXkCCl31lC9oh.png?dl=1" title="QQ截图20191120194049.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24759802/181067108-b474a2fa-49cc-4b46-b083-260585f149dd.png" alt="QQ截图20191120194049.png" data-base62-sha1="kUo95LHenJWgLTjXkCCl31lC9oh" width="650" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/928b4b424c5152e3be22a6924bdbc37a76692345_2_650x500.png, https://aws1.discourse-cdn.com/github/optimized/1X/928b4b424c5152e3be22a6924bdbc37a76692345_2_975x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/928b4b424c5152e3be22a6924bdbc37a76692345.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/928b4b424c5152e3be22a6924bdbc37a76692345_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">QQ截图20191120194049.png</span><span class="informations">1174×903 154 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>When I’m logged in (the pic links are added by myself due to this issue):</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/3939d4155b957c4bae807ad2edd7abf813e44ed4.png" data-download-href="/uploads/short-url/8af6ZdCrNeKQio2sez4ObHMHNoU.png?dl=1" title="QQ截图20191120194322.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24759802/181067109-a2224ad0-9a40-40ce-a8e7-b1d1114b251a.png" alt="QQ截图20191120194322.png" data-base62-sha1="8af6ZdCrNeKQio2sez4ObHMHNoU" width="656" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/3939d4155b957c4bae807ad2edd7abf813e44ed4_2_656x500.png, https://aws1.discourse-cdn.com/github/optimized/1X/3939d4155b957c4bae807ad2edd7abf813e44ed4_2_984x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/3939d4155b957c4bae807ad2edd7abf813e44ed4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/3939d4155b957c4bae807ad2edd7abf813e44ed4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">QQ截图20191120194322.png</span><span class="informations">1253×955 153 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lee-dohm** [comment]
<p>I’m not sure what the problem is you’re asking about? The images you linked in your post don’t give me a good idea of what it is you’re runing into. Perhaps you could give some <a href="https://www.lee-dohm.com/2015/01/04/writing-good-bug-reports/" rel="nofollow noopener">steps to reproduce the problem</a> you’re seeing?</p>

**Comment by imba-tjd** [comment]
<p>Let’s talk about this post.</p>
<p>This is what I got, when I’m not logged in: <a href="https://user-images.githubusercontent.com/24759802/71820998-dc419000-30cb-11ea-8566-36d84d57164e.jpg" rel="nofollow noopener">https://user-images.githubusercontent.com/24759802/71820998-dc419000-30cb-11ea-8566-36d84d57164e.jpg</a></p>
<p>This is what I got when I’m logged in: <a href="https://user-images.githubusercontent.com/24759802/71821005-e499cb00-30cb-11ea-93de-4f91a61c3344.jpg" rel="nofollow noopener">https://user-images.githubusercontent.com/24759802/71821005-e499cb00-30cb-11ea-93de-4f91a61c3344.jpg</a></p>
<p>I can’t see my pictures. Is this expected?</p>

**Comment by lee-dohm** [comment]
<p>No, it isn’t expected. So, if you could give me <strong>the exact steps</strong> to reproduce the problem, including what browser and OS (with versions) you’re using, we’ll take a deeper look.</p>

**Comment by imba-tjd** [comment, marked as answer]
<p>Resolved by unchecking the “ignore HTML” in “user-preferences”. Sorry for the inconvenience.</p>
